cmds/webboot/webboot.go

```diff
@@ -108,3 +113,21 @@ func download(URL, fPath string) error {
 	}
 	return nil
 }
+
+func getCachedIsos(cachedDir string) []*CachedISO {
```

Comment:
At what point is this function called?

Reply:
I think I will call a function to get all the block device, then mount it one by one, and call this function to all  iso under the device and returns them. Then in the main function add these iso to the menu